rest-api/config/config_stable.json

```diff
@@ -9,6 +9,13 @@
       "roles": ["healthpro"],
       "whitelisted_appids": ["pmi-hpo-test"]
     },
+    "awardee-pitt@pmi-drc-api-test.iam.gserviceaccount.com": {
```

Comment:
Is this the SA that's going to get used in stable, too? (Just wondering since it's pmi-drc-api-test)

Reply:
Good point, let me go fix that.